packages/lib-classifier/src/store/TutorialStore/TutorialStore.js

```diff
@@ -23,48 +23,22 @@ const TutorialStore = types
     },
```

Comment:
`get isActiveReferenceValid ()` is no longer being used for anything and can be removed.

Reply:
Good catch!